### ISSUE TITLE
"llama_cpp" module is missing

### ISSUE THREAD

**Issue by ninjasaid2k** [oobabooga/text-generation-webui#992]
### Describe the bug

it seems that the "llama_cpp" module is missing and causing the error when attempting to load the "gpt-x" model. 

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

1. Start the web UI.
2. Select the "gpt-x" model to load.
3. The application throws an error indicating that the "llama_cpp" module is missing.

### Logs

```shell
Starting the web UI...

===================================BUG REPORT===================================
Welcome to bitsandbytes. For bug reports, please submit your error trace to: https://github.com/TimDettmers/bitsandbytes/issues
================================================================================
CUDA SETUP: CUDA runtime path found: C:\Users\user\Downloads\oobabooga-windows\oobabooga-windows\installer_files\env\bin\cudart64_110.dll
CUDA SETUP: Highest compute capability among GPUs detected: 7.5
CUDA SETUP: Detected CUDA version 117
CUDA SETUP: Loading binary C:\Users\user\Downloads\oobabooga-windows\oobabooga-windows\installer_files\env\lib\site-packages\bitsandbytes\libbitsandbytes_cuda117.dll...
The following models are available:

1. gpt-x
2. llama-13b-4bit-128g
3. noting
4. RWKV-4-Raven-14B-v8-Eng-20230408-ctx4096

Which one do you want to load? 1-4

1

Loading gpt-x...
Traceback (most recent call last):
  File "C:\Users\user\Downloads\oobabooga-windows\oobabooga-windows\text-generation-webui\server.py", line 302, in <module>
    shared.model, shared.tokenizer = load_model(shared.model_name)
  File "C:\Users\user\Downloads\oobabooga-windows\oobabooga-windows\text-generation-webui\modules\models.py", line 106, in load_model
    from modules.llamacpp_model_alternative import LlamaCppModel
  File "C:\Users\user\Downloads\oobabooga-windows\oobabooga-windows\text-generation-webui\modules\llamacpp_model_alternative.py", line 9, in <module>
    from llama_cpp import Llama
ModuleNotFoundError: No module named 'llama_cpp'
Press any key to continue . . .
```

### System Info

```shell
Operating System: Windows
GPU Brand: Nvidia
GPU Model: RTX 2070 with Q-Max Design
```

**Comment by Reshiban** [comment]
@ninjasaid2k how did you fixed this ?

**Comment by Reshiban** [comment]
**Fixed:**
The python environnement is directly installed in a folder dedicated to **text-generation-webui** project (and is python310).
This means using pip in a classical cmd will not affect the **text-generation-webui** env (previously I was trying to install a file compiled for python310 on an universal python39).

To fix the _llama\_cpp dependencie problem_, open `micromamba-cmd.bat` which allows to open the env, then type `pip install -r ./text-generation-webui/requirements.txt` to install the dependencies listed in the specified .txt file.

**Comment by ryanniedev** [comment]
> **Fixed:** The python environnement is directly installed in a folder dedicated to **text-generation-webui** project (and is python310). This means using pip in a classical cmd will not affect the **text-generation-webui** env (previously I was trying to install a file compiled for python310 on an universal python39).
> 
> To fix the _llama_cpp dependencie problem_, open `micromamba-cmd.bat` which allows to open the env, then type `pip install -r ./text-generation-webui/requirements.txt` to install the dependencies listed in the specified .txt file.

I got the same error, actually, i tried to pip install quite a lot of modules since the start_windows.bat didn't do his job for all the necessary ones. I tried to run the cmd_windows.bat with the above pip install -r ./text-generation-webui/requirements.txt, and this is what shows up:

 × git fetch -q https://github.com/huggingface/peft 3714aa2fff158fdfa637b2b65952580801d890b2 did not run successfully.
  │ exit code: 128
  ╰─> See above for output.

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× git fetch -q https://github.com/huggingface/peft 3714aa2fff158fdfa637b2b65952580801d890b2 did not run successfully.
│ exit code: 128
╰─> See above for output.

so I tried to run start_windows.bat, the topic error shows up... anyone who can help me here? 

thanks in advance!
